MapTopic:                   '/quadrotor/mapper/local_voxel_no_inflation_map'

TargetTopic:                '/move_base_simple/goal'

DilateRadius:               0.05

VoxelWidth:                 0.10

MapBound:                   [-20, 20, -20, 20, -5, 5]

TimeoutRRT:                 0.2

MaxVelMag:                  3.0

MaxBdrMag:                  2.1

MaxTiltAngle:               1.05

MinThrust:                  2.0

MaxThrust:                  12.0

VehicleMass:                0.61

GravAcc:                    9.8

HorizDrag:                  0.70

VertDrag:                   0.80

ParasDrag:                  0.01

SpeedEps:                   0.0001

WeightT:                    20.0

ChiVec:                     [1.0e+4, 1.0e+4, 1.0e+4, 1.0e+4, 1.0e+5]

SmoothingEps:               1.0e-2

IntegralIntervs:            16

RelCostTol:                 1.0e-5
